.travis.yml

```diff
@@ -25,6 +36,16 @@ install:
   - conda list
   - conda info -a
   - pip install -r requirements.txt
+  - if "$PYSAL_PYPI"; then
+      echo 'testing pypi libpysal esda mapclassify giddy spreg';
+    else
+      echo 'testing git libpysal esda mapclassify giddy spreg';
+      git clone https://github.com/pysal/libpysal.git; cd libpysal; pip install .; cd ../;
```

Comment:
I would replace `git clone ...; cd ;; pip install .` lines with `pip install https://github.com/<org>/,project>/archive/master.zip` Thats a bit shorter and faster.